packages/Python/lldbsuite/test/lang/swift/swift_version/TestSwiftVersion.py

```diff
@@ -0,0 +1,93 @@
+# TestSwiftVersion.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test that LLDB can debug  code generated by the Swift compiler for different versions of the language
+"""
+import commands
+import lldb
+from lldbsuite.test.lldbtest import *
+import lldbsuite.test.decorators as decorators
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import os.path
+import time
+import unittest2
+
+
+def execute_command(command):
+    # print '%% %s' % (command)
+    (exit_status, output) = commands.getstatusoutput(command)
+    # if output:
+    #     print output
+    # print 'status = %u' % (exit_status)
+    return exit_status
```

Comment:
TestExec.py and TestObjCiVarIMP.py have pretty much this same function defined, except that they do:

    (exit_status, output) = seven.get_command_status_output(command)

seven is a wrapper to handle py2 and py3, so you probably want to use it.  And three copies of this is too many, can you move this up into lldbtest or somewhere central and share it?